modules/stardate/tests/test_stardate.c

```diff
@@ -38,9 +38,9 @@ stardate_assert(const gchar *msg_str, const int precision, const gchar *expected
 
   char *template_command;
   if (precision == -1)
-    asprintf(&template_command, "$(stardate $UNIXTIME)");
+    g_assert( asprintf(&template_command, "$(stardate $UNIXTIME)") > -1);
```

Comment:
Please capture the return value into a variable and then `g_assert()` on that, instead.

Also, `>= 0` is perhaps a little more intuitive than `> -1`.

Reply:
Thank you for notice, nice catch! It is okay for you if I just assert for the error case (http://man7.org/linux/man-pages/man3/asprintf.3.html) to avoid segfault? The actual return value is tested in the next line.